onchain/src/social/account.cairo

```diff
@@ -70,12 +86,19 @@ pub mod SocialAccount {
 
             erc20.transfer(request.content.recipient_address, request.content.amount);
         }
+
+        fn is_supported_interface(self: @ContractState, interface_id: felt252) -> bool {
```

Comment:
What is this for? Method `fn supports_interface(felt252) -> bool;` is already provided by the component, no?

Reply:
This is an external function that we use to check if we're supporting a particular interface.
`is_supported_interface` function helps verify that the SocialAccount contract complies with the interface standards.
You can see I'm using `supports_interface` in it.

Reply:
Right.